Source/santactl/Commands/sync/SNTCommandSyncConstants.m

```diff
@@ -34,6 +34,8 @@
 NSString *const kBinaryRuleCount = @"binary_rule_count";
 NSString *const kCertificateRuleCount = @"certificate_rule_count";
 NSString *const kFCMToken = @"fcm_token";
+NSString *const kFCMFullSyncInterval = @"fcm_full_sync_interval";
+NSString *const kFCMGlobalRuleDeadline = @"fcm_global_rule_deadline";
```

Comment:
I know I suggested this but, looking at it again, I think "fcm_global_rule_sync_deadline" is the more clear way of conveying the constant's role. Thoughts?